src/curl.cpp

```diff
@@ -2450,7 +2450,7 @@ string S3fsCurl::CalcSignatureV2(const string& method, const string& strMD5, con
     free(md);
```

Comment:
Here too, it must be modified to `delete[] md`.

Reply:
Done.

Reply:
Thanks!